deployments/helm/KubeArmor/values.yaml

```diff
@@ -15,6 +15,14 @@ kubearmorRelay:
     tag: latest
   # kubearmor-init imagePullPolicy
   imagePullPolicy: Always
+  # Add environment variables for STDOUT logging
+  env:
+    - name: ENABLE_STDOUT_LOGS
+      value: "true"
+    - name: ENABLE_STDOUT_ALERTS
+      value: "true"
+    - name: ENABLE_STDOUT_MSGS
+      value: "true"
```

Comment:
We might want to keep these `false` by default. stdout logging has performance implications.